.gitignore

```diff
@@ -21,4 +21,7 @@ tests/tmp*
 
 .DS_Store
 .idea/
-.vscode
\ No newline at end of file
+.vscode
```

Comment:
Adding editor-specific patterns in `.gitignore` is generally considered bad practice. See [this article](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) on how to create a global `.gitignore` for your machine.